src/tools.cpp

```diff
@@ -91,6 +92,80 @@ StateHistory sampleSGP4Orbit( const Tle& tle,
     return stateHistory;
 }
 
+//! Convergence test for a virtual TLE generated from the cartesian to TLE converter in atom
```

Comment:
//! Execute convergence test for a virtual TLE.

(Should be same as short description in .hpp file.)